src/pharmpy/workflows/model_database/local_directory.py

```diff
@@ -103,26 +103,21 @@ def store_model(self, model):
         model = model.copy()
         model.update_datainfo()
         path = self.path / '.datasets'
-        if not path.is_dir():
-            path.mkdir(parents=True)
-        di = model.datainfo.copy()
+        path.mkdir(parents=True, exist_ok=True)
         for dipath in path.glob('*.datainfo'):
             curdi = DataInfo.read_json(dipath)
-            di.path = curdi.path  # This could be different in comparison
```

Comment:
Not needed anymore. A consequence of changing the behavior of `DataInfo#__eq__`.